apps/cowswap-frontend/src/modules/trade/containers/TradeWidget/TradeWidgetForm.tsx

```diff
@@ -161,6 +162,22 @@ export function TradeWidgetForm(props: TradeWidgetProps) {
                 topLabel={isWrapOrUnwrap ? undefined : inputCurrencyInfo.label}
                 {...currencyInputCommonProps}
               />
+
+              {isLimitOrderMode && !isWrapOrUnwrap && (
+                <InlineBanner
+                  dismissable
+                  bannerID="limitOrders_zeroBalanceBanner"
```

Comment:
Better to version it. And I'm not a fan of `_` in names:

```suggestion
                  bannerID="limitOrdersZeroBalanceBanner:v0"
```

Reply:
Will fix

Reply:
fixed